trade-finance-manager-ui/server/controllers/utilisation-reports/update-utilisation-report-status/index.ts

```diff
@@ -67,12 +69,27 @@ const getReportWithStatus = (reportIdentifier: ReportIdentifier, formButton: str
   }
 };
 
-export const updateUtilisationReportStatus = async (req: Request, res: Response) => {
+type UpdateUtilisationReportStatusRequestBody = {
+  _csrf: string;
+  'form-button': string;
+  'submission-month': string;
+} & {
+  [key: string]: 'on'; // all checkboxes in payload have value 'on'
+};
+
+interface UpdateUtilisationReportStatusRequest extends Request {
+  body: UpdateUtilisationReportStatusRequestBody;
+}
+
+export const updateUtilisationReportStatus = async (req: UpdateUtilisationReportStatusRequest, res: Response) => {
   const { user, userToken } = asUserSession(req.session);
-  const { 'form-button': formButton, reportPeriodStartMonth, reportPeriodYear } = req.query;
 
   try {
-    const reportIdentifiers = getReportIdentifiersFromBody(req.body, Number(reportPeriodStartMonth), Number(reportPeriodYear));
+    const { 'form-button': formButton, 'submission-month': submissionMonth } = req.body;
+    assertValidIsoMonth(submissionMonth);
+    const reportPeriodStart = getReportPeriodStart(submissionMonth);
+
+    const reportIdentifiers = getReportIdentifiersFromBody(req.body, reportPeriodStart.month, reportPeriodStart.year);
```

Comment:
Now that `submissionMonth` is part of `req.body` can this just be
```
const reportIdentifiers = getReportIdentifiersFromBody(req.body);
```
then let that function work everything out?